js/src/react/components/ErrorBlock/ErrorBlock.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import "./styles.css";
+
+export default function ErrorBlock({data, supportLink, supportLinkText}) {
+    const isArray = Array.isArray(data);
+
+    if (isArray) {
+        return (
+            <div className="error">
+                {data.map((error, index) => (
+                    <h4 key={index} style={{wordBreak: 'break-all'}}>{error}</h4>
+                ))}
+                <h4 style={{flex: '1 1 auto', textAlign: 'right'}}>
+                    <span>{supportLinkText}:&nbsp;</span>
+                    <a target="_blank" href={supportLink}>
+                        {supportLink}
+                    </a>
+                </h4>
+            </div>
+        );
+    }
```

Comment:
Need `else` handled here to render `isArray` error maybe.

Reply:
Thanks, i reverted this changes to other branch and forms new pull, there fixed the issue https://github.com/CleanTalk/security-malware-firewall/pull/515/commits/98f7c887f517eaf53a997393be1067fc8d163905